js/build-lists.js

```diff
@@ -1,9 +1,9 @@
+// eslint-disable-next-line no-unused-vars
 var DynamicLists = function(data, container) {
-  var _this = this;
-
   this.data = data;
   this.$container = $(container);
-  
+
   console.log('General: ', data);
```

Comment:
Since this is a PR for linting and cleaning up code, can we remove this `console.log` too please?

Reply:
@tonytlwu Yes, I will remove it. There are other consoles in the LFD. But they almost all showing errors, should we remove them too?

Reply:
I saw them too. We can keep them.

Reply:
@tonytlwu removed console log.